docs: update project-lifecycle.md for grammar (#28)
hyperledger/tsc: project-lifecycle.md

# Project Lifecycle

> The term "project" within Hyperledger refers to a

Author: Informing readers about project lifecycles require some verbs to use future tense. I suggest using present tense when possible, following https://developers.google.com/style/tense?hl=en

collaborative endeavor to deliver a work item.
There may be some projects that are intended to produce a document, such
as a requirements or use cases document, a whitepaper, or analysis.
Other projects develop a new capability, refactor, or remove an
existing capability for the Hyperledger technology releases. Such

> projects may take the form of a new component (for example, a new repository) or

Reviewer: I believe `e.g.,` is okay

> projects may take the form of a new component (for example, a new repository) or

Author: ```suggestion
projects may take the form of a new component (e.g., a new repository) or
```

may propose additions, deletions, or changes to an existing

> repository or repositories.

Author: suggest we adhere to conventions as  https://developers.google.com/style/plurals-parentheses?hl=en

# Proposal

> Project Proposals are submitted to the [TSC](https://www.hyperledger.org/about/leadership)

Reviewer: The verb here could even be `must be`, as this is the only way in which we accept project proposals today.

> Project Proposals are submitted to the [TSC](https://www.hyperledger.org/about/leadership)

Author: ```suggestion
Project Proposals must be submitted to the [TSC](https://www.hyperledger.org/about/leadership)
```

for review, using [Proposal Template](https://hyperledger.github.io/hyperledger-hip/).
Proposals that are approved enter into an *Incubation* state, unless
they are of a refactoring nature, in which case the project is turned over
to the relevant project maintainers to handle as they deem fit.

A Proposal must:

> -   Have a clear description

Author: suggest capitalizing the first letter in a list item, following https://developers.google.com/style/lists?hl=en#numbered-lettered-bulleted-lists

-   Have a well-defined scope
-   Identify committed development resources
-   Identify initial maintainers
-   Be vendor neutral

# Incubation

Approved project proposals enter into *Incubation*. For new
components and modules, a repository is created under the
[Hyperledger Github org](https://github.com/hyperledger)

> and optionally in [JIRA](https://jira.hyperledger.org), if requested. New

Reviewer: ```suggestion
. New
```

We no longer use Jira. Obviously not a grammar issue, but something that we should fix at some point.

> features/capabilities must be handled through pull requests labeled

Author: should is ambiguous

> features/capabilities must be handled through pull requests labeled

Reviewer: ```suggestion
features or capabilities must be handled through pull requests labeled
```
For consistency with other changes you are making.

with tags that identify the project and tag it as
*incubator*. Pull requests ideally are capable of being enabled and disabled with feature-flags.

Projects in *Incubation* can overlap with one another.
Entering *Incubation* is meant to be fairly easy to allow for
community exploration of different ideas.

After a project qualifies to be declared *Graduated*, the
*project* maintainers can then vote to request a graduation
review by the TSC.

Entering *Incubation* does not guarantee that the project will
eventually get to the *Graduated* state. Projects may never get

> to *Graduated* state.

Reviewer: ```suggestion
to the *Graduated* state.
```
Should we add `the` here to be consistent with previous change?

# Dormant

Projects enter the *Dormant* state when the normal functions are

> suspended or slowed down for a period of time. The TSC decides to move project to or from the *Dormant* state upon request.

Reviewer: ```suggestion
suspended or slowed down for a period of time. The TSC decides to move a project to or from the *Dormant* state upon request.
```

If *Dormant* projects are re-activated, they re-enter the *Incubation*
state even if they entered the *Dormant* state from the *Graduated* state.

# Deprecated

Anyone may propose that a project be *Deprecated* by submitting a
rationale and identifying a substitute project or component, if any. The
maintainers of the project shall vote on such a request and if it
passes, make that recommendation to the TSC. Members of the community
that disagree with the request can make their case before the TSC. The

> TSC will consider all points of view and render a final decision to

Reviewer: ```suggestion
TSC will consider all points of view and render a final decision
```
in combination with my suggestion on the next line

> deprecate or not deprecate.

Reviewer: ```suggestion
whether to deprecate.
```
